terraform/ecr-sandbox/root_data.tf

```diff
@@ -0,0 +1,3 @@
+data "aws_ssm_parameter" "cost_centre" {
+  name = "/sandbox/cost_centre"
```

Comment:
Is this parameter already in the sandbox account? Should there be an instruction to add it?

Reply:
Yep, it's there so that other scripts can use it.

It shouldn't be removed, and I think it's easy enough to add it back if it ever does get deleted. Is that enough, do you think? If you tried to run this script and hit an error because this parameter doesn't exist, would you know what to do or does it need some more explanation?